.gitlab-ci.yml

```diff
@@ -314,7 +331,20 @@ build:base+32bit:
     variables:
       - $FULL_CI == "true"
 
+build:no-native:
+  extends: .build-template
+  variables:
+    COQ_EXTRA_CONF: "-native-compiler ondemand"
```

Comment:
Hi, maybe the wording could be improved? (w.r.t. `build:no-native` vs. `-native-compiler ondemand` :)

Reply:
what about this renaming? (but YMMV)
* `build:no-native` → `build:native-ondemand`
* and `build:edge+flambda` → `build:edge+flambda:no-native`